README.md

```diff
@@ -46,17 +46,14 @@ If you are using Mozilla Firefox, then read the [Documentation for geckodriver](
 
 # User Manual
 
-Hi! Need help to be more productive? Good news, I'm here to manage your time! Feel free to ask me stuff :)
```

Comment:
I've added the first name Gaby. So the message looks like this in my Telegram

> Hi, Jason! Need help to be more productive? Good news, I'm here to manage your time! Feel free to ask me stuff!
Want to know me more? Just ask me whatever you want and hope I can understand
Want to know what I can do? Just run /help to see commands that I can do to help you